app/javascript/listings/listingDashboard.jsx

```diff
@@ -109,9 +109,10 @@ export class ListingDashboard extends Component {
         }}
         className={`rounded-btn ${filter === f ? 'active' : ''}`}
         role="button"
-        onKeyPress={event => {
-          event.key === 'Enter' && this.setState({ filter: event.target.textContent });
-        }}
+        onKeyPress={event =>
+          (event.key === 'Enter' || event.key === ' ') &&
+          this.setState({ filter: event.target.textContent })
+        }
```

Comment:
could you use a function for these three event handlers? the only difference is the hash of the state